include/boost/url/rfc/reg_name_rule.hpp

```diff
@@ -36,7 +36,6 @@ struct reg_name_rule
 {
     pct_encoded_str v;
 
-    BOOST_URL_DECL
```

Comment:
it would have been better if these unrelated changes were in their own commit. too late now but something to think about in the future.

Reply:
Yep. That was causing a problem with "GCC" on windows.